scripts/raport-regressions.scala

```diff
@@ -18,12 +18,24 @@ import org.apache.http.auth.*
 import scala.io.Source
 import scala.concurrent.*
 import scala.concurrent.duration.*
-import scala.io.AnsiColor.*
+// import scala.io.AnsiColor.*
 import org.elasticsearch.client.RestClient
 import org.apache.http.HttpHost
 
 given ExecutionContext = ExecutionContext.global
 
+// TODO refactor colors out of file
+val RED = """<span style="color:red">"""
+val YELLOW = """<span style="color:yellow">"""
+val MAGENTA = """<span style="color:magenta">"""
+val RESET = """</span>"""
+val BOLD = """<span style="font-weight:bold">"""
+val LINE_BREAK = "<br>"
+
+def println(text : String): Unit = {
+  System.out.println(s"text $LINE_BREAK")
+}
+
```

Comment:
Can we maybe factor this out? I was thinking that we might have a simple console output for ad-hoc comparison and producing an enriched markdown when targeting the gh page. It might contain some logging capability so can remove the commented-out prinmtlns below